compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -1092,12 +1092,9 @@ ofInterestRule opts = do
                     vrNoteSetNotification ovr $ LF.fileWScenarioNoLongerCompilesNote $ T.pack $
                         fromNormalizedFilePath file
 
-        let dlintEnabled = case optDlintUsage opts of
-              DlintEnabled _ _ -> True
-              DlintDisabled -> False
         let files = HashSet.toList scenarioFiles
         let dalfActions = [notifyOpenVrsOnGetDalfError f | f <- files]
-        let dlintActions = [use_ GetDlintDiagnostics f | dlintEnabled, f <- files]
+        let dlintActions = [use_ GetDlintDiagnostics f | f <- files]
```

Comment:
we don't need to check `dlintEnabled` anymore because `GetDlintDiagnostics` will produce no output when `optDlintUsage = DlintDisabled` (previously it would crash).